vm.go

```diff
@@ -145,6 +147,14 @@ func CreateVirtualMachineSpec(opts *VirtualMachineOptions) (*VirtualMachineSpec,
 		spec.VirtualMachine.Chipset.Uefi.ApplySecureBootTemplate = "Apply"
 	}
 
+	if opts.HighMmioBaseInMB != 0 {
```

Comment:
memory_2.go says API version 2.1 -> but in here it says its applicable for schema 2.3. do we need to change the version?

Reply:
I'm not sure if the api version means the schema version (you can see both memory.go and memory_2.go declare they are on API v2.1).  Unless we want to create a lot of variants classes for each verson schema otherwise I prefer just use one struct to cover the schema version differences.